test/test_images/probe_helper/probe_helper.yaml

```diff
@@ -33,11 +33,23 @@ spec:
         - name: probe-helper
           image: ko://github.com/google/knative-gcp/test/test_images/probe_helper
           env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
as a follow up, we will modify the setup script to configure the clusters with this secret